Automattic/jetpack
# Instant Search: set the default overlay trigger to 'immediate'

Fixes #19881.

#### Changes proposed in this Pull Request:
Set the Instant Search overlay trigger to 'immediate' for new users.

As outlined in #19881, "open when results are available" doesn't appear to often any advantage over "immediate". It can offer a more jarring UX if the modal opens when you are still typing. I suggest we should consider removing it completely in the future.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Create up a new Jetpack site using jurassic.ninja and use Jetpack Beta to switch to this PR's branch.
2. Signup for Jetpack Search for the new site at https://wordpress.com/jetpack-search.
3. Go to Customizer and ensure that the selected overlay trigger is 'immediate'.
4. Ensure you can still select and save other overlay triggers in the Customizer.
